boards/same54-xpro/board.c

```diff
@@ -97,3 +97,13 @@ VFS_AUTO_MOUNT(lwext4, VFS_MTD(sdhc_dev), VFS_DEFAULT_SD(0), 1);
 #endif
 #endif /* MODULE_VFS_DEFAULT */
 #endif /* MODULE_SAM0_SDHC */
+
+#ifdef MODULE_PERIPH_FREQM
+#include "periph/freqm.h"
+    struct freqm_config config_freqm = {
```

Comment:
```suggestion
    const struct freqm_config config_freqm = {
```

If you can make it `const` it will be stored in ROM only